simple/stats/data.py

```diff
@@ -138,6 +139,14 @@ def triples(self) -> list[Triple]:
       triples.append(
           Triple(self.id, _PREDICATE_DESCRIPTION,
                  object_value=self.description))
+
+    # Encode NL sentences as searchDescription triples.
+    for nl_sentence in self.nl_sentences:
```

Comment:
do you want to rename nl_sentences to search_descriptions as well?

Reply:
Deprecated the `nlSentences` config property and using `searchDescriptions` now. Can remove the former completely down the line.